### ISSUE TITLE
Update WebRTC.framework

### ISSUE THREAD

**Issue by tudorpopa39** [nubomediaTI/Kurento-iOS#38]
Hello,

Could you please update the WebRTC.framework file? A new version 61.5.19063 was released soon, and I would like to use inside the KurentoToolbox. 

https://github.com/Anakros/WebRTC

Thank you very much and best regards

**Comment by smolskyaleksey** [comment]
Hi, u can use my fork https://github.com/smolskyaleksey/Kurento-iOS

**Comment by fsp-rts-2** [comment]
@smolskyaleksey hi, im using your fork version of kurento-ios but I keep on getting these 2 error in 'NBMTreeManager.m: 

> No visible @interface for 'NBMWebRTCPeer' declares the selector 'generateOffer:completion:'
and 

> No visible @interface for 'NBMWebRTCPeer' declares the selector 'generateOffer:completion:'

Is there any way to solve it? thank you.

**Comment by smolskyaleksey** [comment]
Yes, sorry. Just delete last commit. Or use pod with explicitly commit. 
Example: 
pod 'Alamofire', :git => 'https://github.com/Alamofire/Alamofire.git', :commit => '0f506b1c45'

**Comment by smolskyaleksey** [comment]
Also you can use new generateoffer method with bool flag NO(FALSE)

**Comment by fsp-rts-2** [comment]
@smolskyaleksey How do i use the new generateoffer method with bool flag NO(FALSE) method? sorry to trouble you but I'm new to objective c

**Comment by smolskyaleksey** [comment]
```obj-c 
[self generateOffer:<#connectionId#> withLocalPeer:NO completion:<#completion(sdpOffer, connection)#>;
```

**Comment by fsp-rts-2** [comment]
@smolskyaleksey i delete the last commit, but it is still showing the same error.

**Comment by smolskyaleksey** [comment]
Can you show what do you do?

**Comment by fsp-rts-2** [comment]
@smolskyaleksey i git reset 9633646 --hard then pod install then i clean and build. But the build got same errors.
![screen shot 2018-04-03 at 4 36 02 pm](https://user-images.githubusercontent.com/37096007/38238594-e37ab050-375c-11e8-9f8e-a32fc5781109.png)

**Comment by smolskyaleksey** [comment]
How did you call  generateOffer method?

**Comment by fsp-rts-2** [comment]
@smolskyaleksey should be this: - 

```
 (void)generateOffer:(NSString *)connectionId completion:(void(^)(NSString *sdpOffer, NBMPeerConnection *connection))block {
    [self generateOffer:connectionId withDataChannels:NO completion:block];
 }
```

**Comment by smolskyaleksey** [comment]
Should works. Please clean and build pods and project

**Comment by fsp-rts-2** [comment]
@smolskyaleksey i clean the whole build folder so now it is working. Thank you. By the way, do you know how to go into a room server? cause when i try, it give this error: 
![screen shot 2018-04-03 at 7 37 17 pm](https://user-images.githubusercontent.com/37096007/38246985-3702d054-3776-11e8-913d-aaa85e627edb.png)

**Comment by Utkarsh3441** [comment]
I am getting the same issue -[NBMRoomManager webRTCPeer:didCreateLocalCapturer:]: unrecognized selector sent to instance.
Searched for the method declaration. Seems missing

**Comment by SmolskyNeklo** [comment]
https://github.com/smolskyaleksey/Kurento-iOS/blob/master/Classes/Room/NBMRoom.h
there is no NBMRoomManager in my classes.

**Comment by smolskyaleksey** [comment]
https://github.com/smolskyaleksey/Kurento-iOS/blob/master/Classes/Room/NBMRoom.h
there is no NBMRoomManager in my classes.

**Comment by smolskyaleksey** [comment]
Look at 
https://github.com/smolskyaleksey/Kurento-iOS/blob/08c99e9f65959911d2bee2fcded89cdd2c60632b/Classes/WebRTC/NBMWebRTCPeer.m#L491

**Comment by smolskyaleksey** [comment]
``` swift
    func startCall(withConfiguration configuration: NBMMediaConfiguration = NBMMediaConfiguration.default()) -> Void {
        let roomURL = NSURL(string: AppUrls.ROOM_SERVER)! as URL
        let room = NBMRoom(username:myNameInRoom, roomName:calleeID, roomURL:roomURL, dataChannels: false)
        let roomClient = NBMRoomClient(room: room, timeout: 15, delegate: self)
        if (roomClient?.connectionState == NBMRoomClientConnectionState.closed) {
            roomClient?.connect()
        }
        let peer = NBMWebRTCPeer(delegate: self, configuration: configuration)
        guard let webRTCManager = peer else {
            let error = NSError(domain: "com.mydomain",
                                code: 0,
                                userInfo: [NSLocalizedDescriptionKey : "Impossible to setup local media stream, check AUDIO & VIDEO permission"])
            self.delegate?.call(self, didFailWithError: error)
            return
        }
        let isStarted = webRTCManager.startLocalMedia()
        self.webRTCManager = webRTCManager
        self.roomClient = roomClient
        if isStarted {
            self.enableVideo(hasVideo)
            self.delegate?.call(self, didAddLocalStream: webRTCManager.localStream)
        }
    }
```

I used it like this

**Comment by Utkarsh3441** [comment]
In NBMWebRTC.h, For  **NBMWebRTCPeerDelegate**protocol  there is a method 
- (void)webRTCPeer:(NBMWebRTCPeer *)peer didCreateLocalCapturer:(RTCCameraVideoCapturer *)localCapturer;

This is being called inside - (RTCVideoTrack *)localVideoTrack {    [_delegate webRTCPeer:self didCreateLocalCapturer:capturer];} method

But there is no definition for the same. Don't know why error console is printing error with NBMRoomManager

**Comment by Utkarsh3441** [comment]
All other method definitions of NBMWebRTCPeerDelegate protocol  are in NBMRoomManager except the missing one which are
- (void)webRTCPeer:(NBMWebRTCPeer *)peer didCreateLocalCapturer:(RTCCameraVideoCapturer *)localCapturer;

- (void)webRTCPeer:(NBMWebRTCPeer *)peer didCreateLocalFileCapturer:(RTCFileVideoCapturer *)fileCapturer;

Any help would be greatly appreciated as i am stuck here for quiet some time now.

**Comment by dohien1109** [comment]
![image](https://user-images.githubusercontent.com/39975930/56140825-a373f080-5f69-11e9-9544-2d00c5e50205.png)
After I pod install, the error is as follows
